2023-04-26 10:03:15,117 INFO    MainThread:61948 [wandb_setup.py:_flush():76] Configure stats pid to 61948
2023-04-26 10:03:15,117 INFO    MainThread:61948 [wandb_setup.py:_flush():76] Loading settings from /Users/milosz/.config/wandb/settings
2023-04-26 10:03:15,117 INFO    MainThread:61948 [wandb_setup.py:_flush():76] Loading settings from /Users/milosz/Documents/Pycharm/InanceAlgo/test_agent/wandb/settings
2023-04-26 10:03:15,117 INFO    MainThread:61948 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-04-26 10:03:15,117 INFO    MainThread:61948 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-04-26 10:03:15,117 INFO    MainThread:61948 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main_test_agent.py', 'program': '/Users/milosz/Documents/Pycharm/InanceAlgo/main_test_agent.py'}
2023-04-26 10:03:15,117 INFO    MainThread:61948 [wandb_init.py:_log_setup():507] Logging user logs to /Users/milosz/Documents/Pycharm/InanceAlgo/test_agent/wandb/run-20230426_100315-1a2lctrh/logs/debug.log
2023-04-26 10:03:15,117 INFO    MainThread:61948 [wandb_init.py:_log_setup():508] Logging internal logs to /Users/milosz/Documents/Pycharm/InanceAlgo/test_agent/wandb/run-20230426_100315-1a2lctrh/logs/debug-internal.log
2023-04-26 10:03:15,117 INFO    MainThread:61948 [wandb_init.py:init():547] calling init triggers
2023-04-26 10:03:15,118 INFO    MainThread:61948 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-04-26 10:03:15,118 INFO    MainThread:61948 [wandb_init.py:init():595] starting backend
2023-04-26 10:03:15,118 INFO    MainThread:61948 [wandb_init.py:init():599] setting up manager
2023-04-26 10:03:15,121 INFO    MainThread:61948 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-04-26 10:03:15,125 INFO    MainThread:61948 [wandb_init.py:init():605] backend started and connected
2023-04-26 10:03:15,128 INFO    MainThread:61948 [wandb_init.py:init():695] updated telemetry
2023-04-26 10:03:15,168 INFO    MainThread:61948 [wandb_init.py:init():732] communicating run to backend with 60.0 second timeout
2023-04-26 10:03:16,539 INFO    MainThread:61948 [wandb_run.py:_on_init():2176] communicating current version
2023-04-26 10:03:16,588 INFO    MainThread:61948 [wandb_run.py:_on_init():2185] got version response 
2023-04-26 10:03:16,588 INFO    MainThread:61948 [wandb_init.py:init():782] starting run threads in backend
2023-04-26 10:03:17,352 WARNING MainThread:61948 [wandb_init.py:init():1167] interrupted
Traceback (most recent call last):
  File "/Users/milosz/opt/anaconda3/envs/Inance/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1145, in init
    run = wi.init()
  File "/Users/milosz/opt/anaconda3/envs/Inance/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 794, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "/Users/milosz/opt/anaconda3/envs/Inance/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/Users/milosz/opt/anaconda3/envs/Inance/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/Users/milosz/opt/anaconda3/envs/Inance/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/Users/milosz/opt/anaconda3/envs/Inance/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/Users/milosz/opt/anaconda3/envs/Inance/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
